hoot-services/Makefile

```diff
@@ -70,6 +71,7 @@ else
 endif
 
 test-all: config
+	echo Running services tests...
```

Comment:
This will add extra text when running with `-s`, I think Makefiles should be as close to silent as we can make them with -s.